workflows/ampliseq.nf

```diff
@@ -528,34 +526,44 @@ workflow AMPLISEQ {
     // SUBWORKFLOW / MODULES : Downstream analysis with QIIME2
     //
     if ( run_qiime2 ) {
-        //Import ASV abundance table and sequences into QIIME2
+        // Import ASV abundance table and sequences into QIIME2
         QIIME2_INASV ( ch_dada2_asv )
         QIIME2_INSEQ ( ch_fasta )
 
-        //Import taxonomic classification into QIIME2, if available
+        // Import phylogenetic tree into QIIME2
+        if ( params.pplace_tree ) {
+            ch_tree = QIIME2_INTREE ( FASTA_NEWICK_EPANG_GAPPA.out.grafted_phylogeny ).qza
+        } else { ch_tree = Channel.empty() }
+
+        // Import taxonomic classification into QIIME2, if available
```

Comment:
I would suggest to move the `tax_agglom_min = params.tax_agglom_min` and `tax_agglom_max = params.tax_agglom_max` assignments here and removing them in the statements below, apart from the cases they are changed.

Reply:
Right, will do!